docusaurus/docs/publish-a-plugin/build-automation.md

```diff
@@ -0,0 +1,61 @@
+---
+id: build-automation
+title: Automate plugin builds
+sidebar_position: 1
+description: Automate Grafana plugin builds and releases
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - automation
+  - build
+  - automate
+  - builds
+---
+
+# Automate packaging and signing with CI
+
+If your plugin has been set up to use the supplied [Github workflows](../create-a-plugin/develop-a-plugin/set-up-github-workflows.md) from [create-plugin](../get-started/get-started.mdx),
+the plugin should be built and packaged in the correct format.
+
+If you need to set up the git worklows, see [these docs](https://github.com/grafana/plugin-actions/blob/main/build-plugin/README.md)
+
+We recommend using the ZIP file produced from this workflow to test the plugin.
+
+If a signing key is included in the Git repo, a signed build is automatically created, which you can use to test the plugin locally before submission.
```

Comment:
This sentence can be confusing, first because git itself doesn't have a mechanism to include secrets (unless is a file) so we should make sure we mention this is for GitHub repository secrets. I suggest we expand it to something like this:

If a Grafana Access Policy Token is included your [Github repository secrets](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-development-environment-secrets-for-your-repository-or-organization), a signed build is automatically created, which you can use to test the plugin locally before submission.

Reply:
much clearer with that, thanks!